client/src/components/content/BpmnModeler/BpmnModeler.jsx

```diff
@@ -67,20 +68,33 @@ const BpmnModeler = (props) => {
       });
   };
 
+  /**
+   * @description Will add a new job with the current robot id to the job list on the server side
+   */
+  const executeBot = () => {
+    const robotId = '6045eccfa9a07940e5763f0b';
```

Comment:
What is it with this warning?

Reply:
It is because robotId is also name of one property of the class. It is irrelevant though because with this [PR](https://github.com/bptlab/ark_automate/pull/174) the function is moving to the bot overview :)